s4e-web/src/app/views/settings/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,29 @@
+import { IBreadcrumb } from './breadcrumb.model';
+import { filter } from 'rxjs/operators';
+import { BreadcrumbService } from './breadcrumb.service';
+import { ActivatedRoute, Router, NavigationEnd } from '@angular/router';
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 's4e-breadcrumb',
+  templateUrl: './breadcrumb.component.html',
+  styleUrls: ['./breadcrumb.component.scss']
+})
+export class BreadcrumbComponent implements OnInit {
+  public breadcrumbs$: Observable<IBreadcrumb[]>;
+
+  constructor(
+    private _router: Router,
+    private _activatedRoute: ActivatedRoute,
+    private _breadcrumbService: BreadcrumbService
+  ) {}
+
+  ngOnInit() {
+    this._router.events
+      .pipe(filter(event => event instanceof NavigationEnd))
```

Comment:
`untilDestoryed` should go somewhere here

And also - you have nested subscription here so some other mechanism to unsubscribe may be required